src/pybind/mgr/cephadm/module.py

```diff
@@ -1746,6 +1746,8 @@ def describe_service(self, service_type=None, service_name=None,
                 n: str = dd.service_name()
                 if service_name and service_name != n:
                     continue
+                if dd.daemon_type == 'osd':
+                    continue                # ignore OSDs for now
```

Comment:
because of the DriveGroup support ... or.. ?

Reply:
yeah.  there is no overall "osd" service spec, only named specs/drivegroups.  and rook doesn't emit this.